java/spacewalk-java.changes

```diff
@@ -1,3 +1,4 @@
+- Validate channel doesn't have cloned channels before delete it(bsc#1138454)
```

Comment:
"Validate that"
"before deleting it (bsc#1138454)"   before and after + -ing

Reply:
Done. Thanks